ios/App/App.xcodeproj/project.pbxproj

```diff
@@ -364,19 +345,18 @@
 			baseConfigurationReference = FC68EB0AF532CFC21C3344DD /* Pods-App.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_STYLE = Manual;
-				DEVELOPMENT_TEAM = "";
+				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
 				INFOPLIST_FILE = App/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				MARKETING_VERSION = 1.0;
 				OTHER_SWIFT_FLAGS = "$(inherited) \"-D\" \"COCOAPODS\" \"-DDEBUG\"";
-				PRODUCT_BUNDLE_IDENTIFIER = "de.fh-muenster.simport.learning";
+				PRODUCT_BUNDLE_IDENTIFIER = de.ifgi.simport.learning;
```

Comment:
Just noticed this change for iOS. Is this intended? Currently the app is existing with the old identifier incl. the certificates in our backend. Can we leave it at `de.fh-muenster.simport.learning` for iOS?

Reply:
This was solved in 9c1fcb045e7106dcf396fa452e6af6a0e3d13d01. Let's see if the iOS build is now working again

Reply:
Certificate is recreated and iOS builds are running again :)